src/content/pt/2019/http2.md

```diff
@@ -0,0 +1,368 @@
+---
+part_number: IV
+chapter_number: 20
+title: HTTP/2
+description: Capítulo de HTTP/2 de 2019 do Web Almanac que cobre a adoção e o impacto do HTTP/2, HTTP/2 Push, Problemas do HTTP/2 e HTTP/3.
+authors: [bazzadp]
+reviewers: [bagder, rmarx, dotjs]
+translators: [elaynelemos]
+analysts: [paulcalvano]
+discuss: 1775
+results: https://docs.google.com/spreadsheets/d/1z1gdS3YVpe8J9K3g2UdrtdSPhRywVQRBz5kgBeqCnbw/
+queries: 20_HTTP_2
+published: 2019-11-11T00:00:00.000Z
+last_updated: 2020-09-21T00:00:00.000Z
+---
+
+## Introdução
+O HTTP/2 foi a primeira grande atualização do principal protocolo de transporte da web em quase 20 anos. Ele chegou com muitas expectativas: prometia um aumento de desempenho gratuito e sem desvantagens. Mais do que isso, poderíamos deixar de lado todos as adaptações e saídas mirabolantes a que o HTTP/1.1 nos forçava, devido às suas ineficiências. Técnicas como bundling, spriting, inlining e até mesmo sharding se tornariam não canônicas em um mundo com HTTP/2, visto que a performance otimizada já seria fornecida por padrão.
+
+Isso significava que mesmo aqueles sem as habilidades e os recursos para se concentrar na [performance na web](./performance) de repente teriam sites com bom desempenho. Contudo, a realidade foi, como sempre, um pouco mais matizada que isso. Já se passaram mais de quatro anos desde a aprovação formal do HTTP/2 como um padrão em maio de 2015 como [RFC 7540](https://tools.ietf.org/html/rfc7540), assim agora é um bom momento para verificar como essa tecnologia relativamente nova se saiu no mundo real.
```

Comment:
```suggestion
Isso significava que mesmo aqueles sem as habilidades e os recursos para se concentrar na [performance na web](./performance) de repente teriam sites com bom desempenho. Contudo, a realidade foi, como sempre, um pouco mais sutil do que isso. Já se passaram mais de quatro anos desde a aprovação formal do HTTP/2 como um padrão em maio de 2015 como [RFC 7540](https://tools.ietf.org/html/rfc7540), assim agora é um bom momento para verificar como essa tecnologia relativamente nova se saiu no mundo real.
```

Just a suggestion to review the nuanced word, I think maybe people will not understand it if the translation is 'matizado'. 

In google, define: nuance
2.
POR ANALOGIA
diferença sutil entre coisas, mais ou menos similares, postas em contraste; matiz, sutileza.